PowerUp/app/build.gradle

```diff
@@ -34,4 +34,8 @@ dependencies {
     androidTestImplementation "com.android.support.test:runner:$rootProject.runnerVersion"
     androidTestImplementation "com.android.support:support-annotations:$rootProject.supportLibraryVersion"
     implementation "com.akexorcist:RoundCornerProgressBar:$rootProject.roundProgressBarVersion"
+    implementation "android.arch.persistence.room:runtime:$rootProject.roomVersion"
+    annotationProcessor "android.arch.persistence.room:compiler:$rootProject.roomVersion"
+
+
```

Comment:
Blank lines here can be removed.

Reply:
+1

Reply:
Idea was to leave one line blank, will update it. Thanks :)